src/subapps/studioLegacy/views/StudioAdminView.tsx

```diff
@@ -4,11 +4,17 @@ import StudioListContainer from '../containers/StudioListContainer';
 import { Link } from 'react-router-dom';
 
 const StudioAdminView: React.FunctionComponent<{}> = () => {
+  // @ts-ignore
```

Comment:
'ts-ignore` 😢

Reply:
ts ignores are due to typescript version upgrade. This does not work with the latest typecsript.  We can remove them before we merge to main.